templates/project/Cargo.toml

```diff
@@ -25,14 +24,14 @@ features = [
 ]
 
 [dependencies]
-embedded-svc = "0.27"
-embedded-hal = { version = "~0.2", features = ["unproven"]}
-log = "0.4"
 async-channel = "2"
+embedded-hal = { version = "~0.2", features = ["unproven"]}
+embedded-svc = "0.27"
 futures-lite = "2"
+log = "0.4"
 
 [build-dependencies]
-cargo_metadata = "0.18"
+cargo_metadata = "0.19"
```

Comment:
Sorting, and some version drift against the ones that are in the worskpace `Cargo.toml` was removed.